lua/lspsaga/finder.lua

```diff
@@ -626,7 +626,21 @@ function Finder:auto_open_preview()
   if not self.short_link[current_line] then
     return
   end
-  local content = self.short_link[current_line].preview or {}
+
+  local content
+  local table_length = 0
+  for key, value in pairs(self.short_link[current_line].preview) do
```

Comment:
there can be `vim.tbl_count(self.short_link[current_line].preview`

Reply:
ok, i will change it